### DISCUSSION TITLE
Mismatch between package release and github code

### DISCUSSION THREAD

**Discussion by pesaventofilippo** [theOehrly/Fast-F1#330] [General]
Latest release on GitHub and PyPI (v2.3.1) has been published 3 days ago, the same date as the latest commit on the `master` branch, so one would assume the codebase should be the same.

Instead, the release package has some errors when fetching data, such as using `.iteritems()` instead of `.items()`, while these problems are already fixed in the `master` branch.

Is there a specific reason the latest release doesn't reflect the latest changes on the repo?

**Comment by theOehrly** [comment]
The master branch is the current development branch for the next major version release v3.0. Originally, there were no further releases planned between v2.3.0 and v3.0. Due to the fact that I am generously missing the intended release date that I had set for myself, it was now necessary to update driver names, teams and colors for the 2023 season. 
I backported the very specific changes to fix this to the branch 'maintenance-v2.3' which the v2.3.1 release is based off. The git tag v2.3.1 points to the correct commit. In general, always refer to the tags when you want to know the state of the repository for a given release.

Depending on how long it actually takes for me to finish v3.0 it might be worth considering fixing things like these annoying pandas warnings as well with another maintenance release. But then of course, that takes time as well that I cannot use to work on v3.0 and time is simply the biggest problem here.

**Reply by pesaventofilippo** [reply]
Got it! Don't worry, great things take time :)

I'll see if I can open a pull request to the maintenance branch to at least fix some of the Pandas warnings.
